### PR TITLE
JMS: Tracker Destination can by specified independent from replyTo Destination with doc

### PR DESCRIPTION
**Pull request by opensource21**
Fix the comments from #3401, adjust the code, so that at can be merged into master and added documentation.
The motivation for the pull request can be found in detail at #3401.

### CODE REVIEW

File: src/sphinx/cheat-sheet.html [hunk 1]
```diff
@@ -3048,7 +3048,7 @@ <h3>Start</h3>
               <h3>Commons</h3>
 
               <div class="function">
-                <label class="function-syntax" for="reqreply">reqreply</label>
+                <label class="function-syntax" for="reqreply">requestReply</label>
```
Comment (slandelle) [inline src/sphinx/cheat-sheet.html]: Woops, nice catch, thanks!

**Comment by slandelle** [comment]
Thanks!

Cherry-picked as 7b52740

**Comment by opensource21** [comment]
Are there any descriptions how to build the download-zip file? Or exists a specific zip-file?

**Comment by slandelle** [comment]
https://gatling.io/docs/current/developing_gatling/building_gatling/

**Comment by opensource21** [comment]
Thanks for the information. Is it correct that the conf-folder is missing and zinc is no longer in a special folder in lib?

**Comment by slandelle** [comment]
> Is it correct that the conf-folder is missing

Right, I think you need to run sbt package in gatling to have it generated

> zinc is no longer in a special folder in lib

yes

**Pull request closed without merging**